lua/nvim-treesitter/shell_command_selectors.lua

```diff
@@ -169,22 +173,38 @@ function M.select_download_commands(repo, project_name, cache_folder, revision)
       M.select_install_rm_cmd(cache_folder, project_name..'-tmp')
     }
   else
+    local git_folder = utils.join_path(cache_folder, project_name)
+    local clone_error = 'Error during download, please verify your internet connection'
+    if is_windows then
+      clone_error = clone_error .. ". If on Windows you may need to enable Developer mode"
```

Comment:
This developer mode was a bit off in the other PR. Wasn't this developer mode only to have the symlinks in Git but actually nobody needs the symlinks? Only the corpus is symlinked. The symlinks were only a problem because tar would crash because of them. For me, L178-180 are not really needed. First, because based on the discussion in the previous PR git won't error and second because this is probably not the likely cause why a download might have failed on Windows (also I wouldn't know how this dev mode can be enabled).

> After some investigation: it appears that git does not resolve symlinks either (by default). It just creates a file containing the path to the file it is linking as text. Git doesn't error, but I assume it caused typescript treesitter to misbehave.

Reply:
https://blogs.windows.com/buildingapps/2016/12/02/symlinks-windows-10/#oTepts0M8148xRoj.97

The reason seems to be that with developer mode you don't need Admin permissions to create symlinks.

Reply:
> also I wouldn't know how this dev mode can be enabled

Update and Security/or developers, second option 😃 
I can try if this setting solves the problem.

But I'm not sure if I like that solution tbh.
"A 3rd party package is probably using symlinks in its repository, please turn on developer mode to install your desired parser" ... sounds a little strange to me. 😉 

As this only happens on updates, maybe we can instead (or as an alternative) tell the user to start neovim in an elevated shell for the update?

Reply:
> '-c', 'core.symlinks=true'

Why? It only causes problems. For the user it's absolutely irrelevant whether symlinks work or not. So why tell them to look for dev mode when it has absolutely no benefit for them?

Reply:
I just thought because of your quote in the first comment:

> Git doesn't error, but I assume it caused typescript treesitter to misbehave.

Reply:
> Git doesn't error,

so it does not make sense to check for the error

>     but I assume it caused typescript treesitter to misbehave.

That was just bullshit. One reason why the other PR was never merged was because he wanted to make symlinks work on Windows while breaking stuff for other users.

This is the symlink in the typescript repo:

https://github.com/tree-sitter/tree-sitter-typescript/blob/master/typescript/corpus/common

the file is absolutely not necessary to compile the parser but is just part of their test suite.

Reply:
I will remove the dev-mode message then.